lib/src/main/scala/spinal/lib/bus/amba3/ahblite/AhbLite3.scala

```diff
@@ -138,7 +138,7 @@ case class AhbLite3(config: AhbLite3Config) extends Bundle with IMasterSlave {
   //return true when the current transaction is the last one of the current burst
   def last(): Bool = {
     val beatCounter        = Reg(UInt(4 bits)) init(0)
-    val isLast             = Vec(U"0000", U"0011", U"0111", U"1111")(U(HBURST >> 1)) === beatCounter || (HREADY && ERROR)
+    val isLast             = Vec(U"0000", U"0011", U"0111", U"1111")(U(HBURST >> 1)) === beatCounter || ERROR
```

Comment:
`isLast` doesn’t indicate properly a burst of type `INCR`, i.e. a burst of undefined length. In its current form, it takes a burst of type `INCR` just the same like the type `SINGLE` which is quite the opposite of `INCR`.

However, see my other note on using `last()` in the AhbLite3Arbiter for why I see this method useless in the first place.